corehq/apps/userreports/management/commands/async_rebuild_table.py

```diff
@@ -29,6 +30,8 @@ def add_arguments(self, parser):
                             help='Only retrieve from one database')
         parser.add_argument('--initiated-by', action='store', required=True, dest='initiated',
                             help='Who initiated the rebuild')
+        parser.add_argument('--start_date', type=date_type,
+                            help='Rebuild on forms received or cases updated on or after this date')
```

Comment:
nit: could be good to specify format

Reply:
https://github.com/dimagi/commcare-hq/pull/26731/commits/3d2012af09d31e442d6f996ace941d25b544e30a